qiskit/utils/deprecation.py

```diff
@@ -37,15 +37,27 @@ def deprecate_arguments(
         Callable: The decorated callable.
     """
 
-    del since  # Will be used in a followup to add deprecations to our docs site.
-
     def decorator(func):
+        func_name = func.__qualname__
+        old_kwarg_to_msg = {}
+        for old_arg, new_arg in kwarg_map.items():
+            msg_suffix = (
+                "will in the future be removed." if new_arg is None else f"replaced with {new_arg}."
```

Comment:
I am not a native speaker, but "will be removed in the future" sounds more natural to me than "will in the future be removed". If there's any specific reason for this order, feel free to ignore my suggestion.

```suggestion
                "will be removed in the future." if new_arg is None else f"replaced with {new_arg}."
```

Reply:
Thanks! No particular reason for the old wording. I intentionally didn't change the wording because this helper is going to be deprecated in https://github.com/Qiskit/qiskit-terra/pull/9676, so it doesn't really matter. This PR is mostly just a refactoring.

But there's little downside to improving this wording also.